definitions/tools/filter_cle_variants.cwl

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env cwl-runner
+ 
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Adds an INFO tag (CLE) to variants in the target file present in a CLE-validated vcf file"
+
```

Comment:
Could we make this name a bit more general, since I can envision other uses for it (looking for specific variants in recurrence samples, etc)?  Maybe call it "filter_existing_variants.cwl" and instead of labelling with the "CLE" tag, use "EXISTING" (or something comparable).    Update: Jason suggests "previously_discovered"

Reply:
Done in 8b52093